File: packages/expo-network/android/src/main/java/expo/modules/network/NetworkPackage.java
package expo.modules.network;

import android.content.Context;

import expo.modules.core.BasePackage;
import expo.modules.core.ExportedModule;

import java.util.Collections;
import java.util.List;

public class NetworkPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new NetworkModule(context));
  }
}

File: packages/expo-network/android/src/main/java/expo/modules/network/NetworkPackage.kt
package expo.modules.network

import android.content.Context

import expo.modules.core.BasePackage
import expo.modules.core.ExportedModule

class NetworkPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> =
    listOf(NetworkModule(context))
}
